sql/updates/world/6.x/2016_05_13_01_world.sql

```diff
@@ -0,0 +1,151 @@
+-- Tol Barad misc
```

Comment:
Yeah I know, I will join them. Left it this way for others to make it more clear to see.